docker/README.md

```diff
@@ -0,0 +1,20 @@
+## Docker related information
+
+This dockerfile sets up the dependencies required to run consensus-spec tests. The docker image can be locally built with:
+- `docker build ./ -t $IMAGE_NAME -f ./docker/Dockerfile`
+
+
+Handy commands:
+- `docker run -it $IMAGE_NAME /bin/sh` will give you a shell inside the docker container to manually run any tests
+- `docker run  $IMAGE_NAME make citest` will run the make citest command inside the docker container
+
+Ideally manual running of docker containers is for advanced users, we recommend the script based approach described below for most users.
+
+The `scripts/build_run_docker_tests.sh` script will cover most usecases. The script allows the user to configure the fork(altair/bellatrix/capella..), image_name(specifies the container to use), number of cores, preset type(mainnet/minimal) and test all forks flags. Ideally this is the main way that users interact with the spec tests, instead of running it locally with varying versions of dependencies.
```

Comment:
```suggestion
The `scripts/build_run_docker_tests.sh` script will cover most usecases. The script allows the user to configure the fork(altair/bellatrix/capella..), `$IMAGE_NAME` (specifies the container to use), number of cores, preset type (mainnet/minimal), and test all forks flags. Ideally, this is the main way that users interact with the spec tests instead of running it locally with varying versions of dependencies.
```